### ISSUE TITLE
Conda create not working for bactopia 3.0.0 

### ISSUE THREAD

**Issue by JLHalpin** [bactopia/bactopia#493]
It happily goes about creating the environment but then doesn't put anything in it.  Someone else in my office was able to use create for bactopia today, so I'm not sure what's going on.  

```
> conda create -n bactopia -c bioconda -c conda-forge -c defaults
Collecting package metadata: done
Solving environment: done

## Package Plan ##

  environment location: /path/.conda/envs/bactopia



Preparing transaction: done
Verifying transaction: done
Executing transaction: done
#
# To activate this environment, use
#
#     $ conda activate bactopia
#
# To deactivate an active environment, use
#
#     $ conda deactivate
```

If I activate this environment and then use conda list to see the installed packages, there are none:
```
> conda list
# packages in environment at /path/.conda/envs/bactopia:
#
# Name                    Version                   Build  Channel
```

**Comment by rpetit3** [comment]
Hi @JLHalpin 

Tack on a `bactopia` at the end of your command, also I recommend putting conda-forge before bioconda:

```
conda create -n bactopia -c conda-forge -c bioconda -c defaults bactopia
```

**Comment by JLHalpin** [comment]
*facepalm*

Thank you!

**Comment by rpetit3** [comment]
Haha no worries! been there plenty of time!

**Comment by rpetit3** [comment]
Hi @JLHalpin 

I've released [Bactopia v3.0.1](https://bactopia.github.io/latest/changelog/#v301-bactopiabactopia-thats-my-girl-20240325), if you give it a try let me know if you have any issues or feedback!

I've also put together a long overdue survey that would be super useful for getting feedback and helping guide future directions of Bactopia. If you have time, I would be super thankful is you completed my [2024 Bactopia Usage Survey](https://forms.gle/BA2BDHRQvZtpVcD38)!

Thank you so much!
Robert

**Comment by JLHalpin** [comment]
Thanks so much!  I did try the new version and it worked great!
